oas_docs/overlays/kibana.overlays.shared.yaml

```diff
@@ -4,6 +4,17 @@ info:
   title: Overlays that are applicable to both serverless and non-serverless documentas
   version: 0.0.1
 actions:
+# Add some tag descriptions and displayNames
+  - target: '$.tags[?(@.name=="maintenance-window")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        You can schedule single or recurring maintenance windows to temporarily reduce rule notifications.
+        For example, a maintenance window prevents false alarms during planned outages.
+      externalDocs:
+        description: Maintenance window documentation
+        url: https://www.elastic.co/docs/explore-analyze/alerts-cases/alerts/maintenance-windows
```

Comment:
Should we have a dedicated URL outside alerts? Like `https://www.elastic.co/docs/explore-analyze/maintenance-windows.` I am not sure, tbh what this URL is about 🙂.